src/EditorFeatures/Test/Diagnostics/DiagnosticAnalyzerServiceTests.cs

```diff
@@ -1084,6 +1084,51 @@ internal async Task TestOnlyRequiredAnalyzerExecutedDuringDiagnosticComputation(
             Assert.Empty(diagnosticMap.Other);
         }
 
+        [Theory, WorkItem(67257, "https://github.com/dotnet/roslyn/issues/67257")]
+        [CombinatorialData]
+        public async Task TestFilterSpanOnContextAsync(bool testSyntaxTreeAction)
```

Comment:
>TestFilterSpanOnContextAsync [](http://example.com/codeflow?start=26&length=28)

Just curious: What is the difference between this test and `DiagnosticAnalyzerTests.TestFilterSpanOnContextAsync`?

Reply:
This tests the IDE change in this PR to pass in the filter span to the call into `CompilationWithAnalyzers.GetAnalysisResultAsync`